istio/istio.spec

```diff
@@ -39,8 +40,11 @@ Source0:        https://%{provider_prefix}/archive/%{git_commit}/%{repo}-%{git_c
 Source1:        istiorc
 Source2:        buildinfo
 
+# CVE patches
+Patch2:         0001-CVE-2020-16844.patch
```

Comment:
Why are we patching it here instead of applying the changes in the repo directly?